inst/include/dust2/cpu.hpp

```diff
@@ -90,6 +90,20 @@ class dust_cpu {
     std::copy_n(it, state_.size(), state_.begin());
   }
 
+  template <typename It>
```

Comment:
I wonder, as I get into these, if `Iter` and `iter` might be clearer names - or perhaps something more about the purpose of the `it` on 94 - not sure.

Reply:
see c1a68b58476ddcc7d1830b308c3c227d92bfb150 now